### PR TITLE
[dashboard] migration cleanups

### PR DESCRIPTION
**Pull request by svenefftinge**
## Description

Some post-migration cleanups in dashboard

## Related Issue(s)

Fixes WEB-469

## How to test

 - chedk

#### Preview status

<p>Gitpod was successfully deployed to your preview environment.</p>
<ul>
	<li><b>🏷️ Name</b> - se-dashboad875e86ef6</li>
	<li><b>🔗 URL</b> - <a href="https://se-dashboad875e86ef6.preview.gitpod-dev.com/workspaces" target="_blank">se-dashboad875e86ef6.preview.gitpod-dev.com/workspaces</a>.</li>
	<li><b>📚 Documentation</b> - See our <a href="https://www.notion.so/gitpod/6debd359591b43688b52f76329d04010#7c1ce80ab31a41e29eff2735e38eec39" target="_blank">internal documentation</a> for information on how to interact with your preview environment.</li>
</ul>

## Build Options

<details>
<summary>Build</summary>

- [ ] /werft with-werft
      Run the build with werft instead of GHA
- [ ] leeway-no-cache
- [ ] /werft no-test
      Run Leeway with `--dont-test`
</details>

<details>
<summary>Publish</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer</summary>

- [ ] analytics=segment
- [ ] with-dedicated-emulation
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

<details>
<summary>Preview Environment</summary>

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [x] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [ ] with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
</details>

/hold

### CODE REVIEW

**Comment by geropl** [comment]
Scanning the code :eyes:

File: components/dashboard/src/app/AppRoutes.tsx [hunk 1]
```diff
@@ -156,7 +154,6 @@ export const AppRoutes = () => {
                     <Route path={usagePathMain} exact component={Usage} />
                     <Route path={settingsPathIntegrations} exact component={Integrations} />
                     <Route path={settingsPathNotifications} exact component={Notifications} />
-                    <Route path={settingsPathBilling} exact component={Billing} />
```
Comment (geropl) [inline components/dashboard/src/app/AppRoutes.tsx:159]: I think we should have a redirect here, for both `/billing` and settingsPathBilling (if they aren't identical) :+1:

File: components/dashboard/src/components/BillingAccountSelector.tsx [hunk 2]
```diff
@@ -1,138 +0,0 @@
...
-import { useContext, useEffect, useState } from "react";
-import SelectableCardSolid from "../components/SelectableCardSolid";
-import { OrganizationInfo, useOrganizations } from "../data/organizations/orgs-query";
-import { ReactComponent as Spinner } from "../icons/Spinner.svg";
-import { getGitpodService } from "../service/service";
-import { UserContext } from "../user-context";
-import Alert from "./Alert";
-import { Subheading } from "./typography/headings";
-
-export function BillingAccountSelector(props: { onSelected?: () => void }) {
```
Comment (geropl) [inline components/dashboard/src/components/BillingAccountSelector.tsx:18]: Soooo glad this is gone :smile:

File: components/dashboard/src/data/organizations/orgs-query.ts [hunk 3]
```diff
@@ -92,8 +92,7 @@ export function useCurrentOrg(): { data?: OrganizationInfo; isLoading: boolean }
         orgId = orgIdParam;
     }
     let org = orgs.data.find((org) => org.id === orgId);
-    if (!org && user?.additionalData?.isMigratedToTeamOnlyAttribution) {
```
Comment (geropl) [inline components/dashboard/src/data/organizations/orgs-query.ts:95]: @svenefftinge No hard dependency on this code change, but: Did you check the prod DB whether this flag is set for all users?

Comment (svenefftinge) [reply components/dashboard/src/data/organizations/orgs-query.ts:95]: yes

File: components/dashboard/src/app/AppRoutes.tsx (hunk 1 above)
Comment (geropl) [reply components/dashboard/src/app/AppRoutes.tsx:159]: More specifically: `/user/billing` -> `/billing` would be nice.
Or, at least have it not stuck in a "Hang in there".

File: components/dashboard/src/admin/UserDetail.tsx [hunk 4]
```diff
@@ -19,42 +19,21 @@ import { getGitpodService } from "../service/service";
...
-    const [creditNote, setCreditNote] = useState<{ credits: number; note?: string }>({ credits: 0 });
-    const [editAddCreditNote, setEditAddCreditNote] = useState<boolean>(false);
     const [editFeatureFlags, setEditFeatureFlags] = useState(false);
     const [editRoles, setEditRoles] = useState(false);
     const userRef = useRef(user);
 
     const initialize = () => {
         setUser(user);
-        const attributionId = AttributionId.render(AttributionId.create(user));
-        getGitpodService().server.adminGetBillingMode(attributionId).then(setBillingMode);
```
Comment (geropl) [inline components/dashboard/src/admin/UserDetail.tsx:45]: Why no longer show BillingMode? I think that should stay here. :upside_down_face:

Comment (geropl) [reply components/dashboard/src/admin/UserDetail.tsx:45]: Ah sorry, ma bad. Ignore.

Comment (geropl) [reply components/dashboard/src/admin/UserDetail.tsx:45]: :thought_balloon: A list of orgs a user is member of would be nice, in the future.

File: components/dashboard/src/admin/WorkspaceDetail.tsx [hunk 5]
```diff
@@ -166,30 +167,33 @@ export default function WorkspaceDetail(props: { workspace: WorkspaceAndInstance
...
+                                </span>
+                                <div className="w-4/12">{wsi.id}</div>
+                                <div className="w-2/12">{dayjs(wsi.startedTime).fromNow()}</div>
+                                <div className="w-2/12">
+                                    {dayjs.duration(dayjs(wsi.stoppingTime).diff(wsi.startedTime)).humanize()}
+                                </div>
+                                <div className="w-2/12">
+                                    {attributionId && attributionId?.kind === "team" ? (
+                                        <Link
+                                            className="text-blue-400 dark:text-blue-600 hover:text-blue-600 dark:hover:text-blue-400"
```
Comment (geropl) [inline components/dashboard/src/admin/WorkspaceDetail.tsx:185]: That's nice. :sparkles:

**Review by geropl — APPROVED** [review]
Tested and works :heavy_check_mark:

**Pull request merged**